### ISSUE TITLE
[BUG]  Inquiry Regarding Pending Registration Status of My Tool

### ISSUE THREAD

**Issue by mohamed1234567890vv** [MscrmTools/XrmToolBox#1322]
Dear @MscrmTools ,

I hope this message finds you well. I am writing to inform you that I have registered my tool on your platform since last Thursday, and it is currently in a pending state. I would like to inquire about the status of my registration and kindly request any updates or additional information regarding the process.

I understand that you may have a high volume of registrations and inquiries to handle, but I would greatly appreciate your attention to my request. Please let me know if there are any further steps or actions required from my side to expedite the process.
![image](https://github.com/MscrmTools/XrmToolBox/assets/30890030/df4567af-6ec0-454a-9712-b9c455678dff)

Thank you for your time and assistance. I look forward to hearing from you soon.

**Comment by MscrmTools** [comment]
The review is a manual process and I was off from Friday to Sunday, then got lot of work today.
I will review your tool again tomorrow

**Comment by BetimBeja** [comment]
We've all been there 😄 The validation process is done manually by @MscrmTools and depends mainly in his availability to deliver this free of charge community work.

Please be patient and respectful 😉

**Comment by MscrmTools** [comment]
yes, please be patient. I already review your tool once so you know I'm aware I have a review to process

**Comment by mohamed1234567890vv** [comment]
Thank you for your response. I appreciate your patience and understanding regarding the registration process for my tool.

I completely understand that you have a review to process and that it may take some time. I assure you that I will patiently await the completion of the review, knowing that you are aware of my registration.

Please let me know if there are any additional information that you require from my end to expedite the process. I am ready to provide any necessary assistance.

Once again, thank you for your attention to this matter. I look forward to hearing from you regarding the status of my registration.

**Comment by mohamed1234567890vv** [comment]
> yes, please be patient. I already review your tool once so you know I'm aware I have a review to process

Thank you for your response. I appreciate your patience and understanding regarding the registration process for my tool.

I completely understand that you have a review to process and that it may take some time. I assure you that I will patiently await the completion of the review, knowing that you are aware of my registration.

Please let me know if there are any additional information that you require from my end to expedite the process. I am ready to provide any necessary assistance.

Once again, thank you for your attention to this matter. I look forward to hearing from you regarding the status of my registration.

**Comment by mohamed1234567890vv** [comment]
> The review is a manual process and I was off from Friday to Sunday, then got lot of work today. I will review your tool again tomorrow

Thank you for your response. I appreciate your patience and understanding regarding the registration process for my tool.

I completely understand that you have a review to process and that it may take some time. I assure you that I will patiently await the completion of the review, knowing that you are aware of my registration.

Please let me know if there are any additional information that you require from my end to expedite the process. I am ready to provide any necessary assistance.

Once again, thank you for your attention to this matter. I look forward to hearing from you regarding the status of my registration.
